src/settings-ui/Microsoft.PowerToys.Settings.UI.Library/ISettingsUtils.cs

```diff
@@ -11,6 +11,9 @@ public interface ISettingsUtils
         T GetSettings<T>(string powertoy = "", string fileName = "settings.json")
```

Comment:
It should not be private as we use both methods. `GetSettings` is used by Image Resizer and `GetSettingsOrDefault` by other modules. Before that change, `GetSettings` returned the default settings object if it can not be retrieved from the settings file. And `ImageResizerSettings` can not be constructed inside of generic method as it requires localized strings.